content/client-lib-development-guide/features.textile

```diff
@@ -381,15 +383,15 @@ h3(#realtime-channel). Channel
 *** @(RTL6g1)@ When publishing a @Message@ with the @clientId@ attribute set to @null@:
 **** @(RTL6g1a)@ It is unnecessary for the client to set the @clientId@ of the @Message@ before publishing
-**** @(RTL6g1b)@ Ably is expected to automatically assign a @clientId@ upon receiving the @Message@. A test should assert that the @clientId@ value is populated for the @Message@ when received
-*** @(RTL6g2)@ When publishing a @Message@ with the @clientId@ attribute value set to the authenticated client's @clientId@, Ably will accept the message and publish it. A test should assert that the @clientId@ value is populated for the @Message@ when received
-*** @(RTL6g3)@ When publishing a @Message@ with a different @clientId@ attribute value to the authenticated @clientId@, the client library should reject publishing immediately. The message should not be sent to Ably and the errr callback should be called. The connection and channel must remain available for further publishing and other operations
-*** @(RTL6g4)@ When using token auth, unless a @clientId@ has been provided in @ClientOptions@ or inferred following authentication, the client library will not be constrained when publishing messages with any explicit @clientId@. If a @Message@ with a @clientId@ value is published before the @clientId@ is configured or inferred following authentication, the client library should not reject any explicit @clientId@ specified in a message. A test should instance a library without an explicit @clientId@ and an @authCallback@ that returns a @tokenDetails@ object with a @clientId@, then publish a message with the same @clientId@ before authentication, and ensure that the message is published following authentication and received back with the @clientId@ intact. A further test should follow the same sequence of events, but should instead use an incompatible @clientId@ in the message, expecting that the message is rejected by the Ably service and the message error should contain the server error message, and the connection and channel should remain available for further operations
+**** @(RTL6g1b)@ Ably will not assign a @clientId@ upon receiving the @Message@. A test should assert that the @clientId@ value is @null@ for the @Message@ when received
+*** @(RTL6g2)@ When publishing a @Message@ with the @clientId@ attribute value set to the identified client's @clientId@, Ably will accept the message and publish it. A test should assert that the @clientId@ value is populated for the @Message@ when received
+*** @(RTL6g3)@ When publishing a @Message@ with a different @clientId@ attribute value to the identified client's @clientId@, the client library should reject publishing immediately. The message should not be sent to Ably and the errr callback should be called. The connection and channel must remain available for further publishing and other operations
```

Comment:
> different @clientId@ attribute value to the identified

different .... from ....

> should reject publishing immediately

I think  "... should reject that Message ..." or "... should reject that publish operation ...". The current wording sounds like perhaps no further messages will be published.